### PR TITLE
Nukes the hundred charges ultime hugbox crime-stopper galore.

### PR DESCRIPTION
**Pull request by Ghommie**
I see no need of a pseudo administrative 100 shots taser for a more relaxed setting without too much mil-sim law indoctrination. From 100 to 20 (formerly 8, requested by many). 
EDIT: Slight accuracy/scatter buff, plus wielded state bonuses.

:cl: Ghommie
balance: Balances tasers, down to 20 shots from the former 100 (yessir, one, zero, zero). Improved accuracy and scatter, both wielded and unwielded. The bolts range has been reduced.
bugfix: Fixes tasers' skill check being broken.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/updated_projectiles/guns/energy.dm [hunk 1]
```diff
@@ -241,4 +240,4 @@
 	if(!isYautja(user))
 		to_chat(user, "<span class='warning'>You have no idea how this thing works!</span>")
 		return
-	return ..()
\ No newline at end of file
+	return ..()
```
Comment (spookydonut) [inline code/modules/projectiles/updated_projectiles/guns/energy.dm]: scream emoji

Comment (Ghommie) [reply code/modules/projectiles/updated_projectiles/guns/energy.dm]: Something wrong? :scream:

File: code/modules/projectiles/updated_projectiles/guns/energy.dm [hunk 2]
```diff
@@ -70,19 +70,18 @@
 	origin_tech = "combat=1;materials=1"
 	matter = list("metal" = 2000)
 	ammo = /datum/ammo/energy/taser
-	movement_acc_penalty_mult = 1
-	charge_cost = 100
+	charge_cost = 1250
```
[the remark this answers is not in the document]
Comment (Ghommie) [reply code/modules/projectiles/updated_projectiles/guns/energy.dm]: I'd go for 500 at most even without a rework, the bolts are still faster than the hyperzined, plus the spread is getting reduced. I'll just reduce the range so people don't get accidentally scoped from two/three screens away.

File: code/modules/projectiles/updated_projectiles/guns/energy.dm (hunk 1 above)
Comment (Ghommie) [reply code/modules/projectiles/updated_projectiles/guns/energy.dm]: the more I think, the more I'm confused on which one gives the early return.

**Pull request merged**